test/IRGen/variadic_generics.sil

```diff
@@ -106,3 +106,31 @@ bb0(%pack : $*Pack{Int, repeat each T, Int}, %value : $Int):
   %ret = tuple ()
   return %ret : $()
 }
+
+sil @blackhole : $<T> (@in T) -> () {
```

Comment:
Making this `@in` instead of `@inout` is technically an ownership violation on both sides of this call (the callee doesn't actually consume the value, and the caller shouldn't be consuming out of an element of an `inout` pack without replacing it).  Someday I hope we'll verify that in the SILVerifier, but even in the meantime, it's probably best to try to be correct about it.

Reply:
Yeah, in OSSA that is verified.  Changed to `@in_guaranteed`.